modules/claims_api/spec/requests/v2/veterans/disability_compensation_request_spec.rb

```diff
@@ -2239,13 +2235,13 @@ def update_json_and_submit(updated_json_lambda)
             context 'has invalid characters in it' do
               let(:treated_center_city) { 'LMNOP 6^7' }
```

Comment:
Should we change this test name to something like `has valid special characters in it`? `invalid` seems misleading.